nais.yaml

```diff
@@ -20,6 +20,8 @@ fasitResources:
       resourceType: credential
     - alias: dagpenger-regel-api-rest
       resourceType: restservice
+    - alias: dagpenger-inntekt-api-rest
```

Comment:
Mtp på at vi skal over på naiserator som ikke støtter fasit, at disse tjenestene er våre og for å unngå ekstra nettverkshopp kan url endres til: http://app.